dojo/models.py

```diff
@@ -688,12 +688,29 @@ def findings_count(self):
             # ideally it's always prefetched and we can remove this code in the future
             self.active_finding_count = Finding.objects.filter(mitigated__isnull=True,
                                             active=True,
+                                            verified=False,
```

Comment:
I think we should stick to the plan we had and active just means active, regardless of verified. i.e. the definitions in https://github.com/DefectDojo/django-DefectDojo/blob/dev/dojo/finding/views.py